### ISSUE TITLE
Error when opening a Dialog from Chip

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#2956]
**Describe the bug**
When clicking the remove icon of the Chip and a Dialog window is rendered, the following error is thrown:

```
Error: ExpressionChangedAfterItHasBeenCheckedError: 
Expression has changed after it was checked. 
Previous value: 'k-chip-focused: true'. 
Current value: 'k-chip-focused: false'.
``` 

**To Reproduce**
1. Click the _edit_ icon of the Chip.
https://stackblitz.com/edit/angular-7gfsz1-vr8nea?file=app/app.component.ts

Reported in support thread: **1470287**

**Comment by siyka-marinova** [comment]
Fixed in the latest [development build](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of `kendo-angular-buttons`.

**Comment by elena-gancheva** [comment]
The fix is available in the `v5.4.4` of the `kendo-angular-buttons` package.
